### ISSUE TITLE
Steam is just a black square in i3 (i3wm)

### ISSUE THREAD

**Issue by techmouse** [ValveSoftware/steam-for-linux#10674]
#### Your system information
* Steam client version (build number or date): No idea. I can't see the menu option to find that information and `steam --version` does nothing. But it's not automatically updating at startup, so I'm assuming I'm at the most recent version.
* Distribution (e.g. Ubuntu): KDE Neon (but using i3wm)
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/14770768/steam-logs.tar.gz)
* GPU: Nvidia GTX 970

#### Please describe your issue in as much detail as possible:
KDE Neon rolled out Plasma 6 and it broke everything for me so I'm migrating to a more proper tiling window manager.

So far, Steam is the only thing not working in i3. I've restarted i3 and Steam in a number of different ways but I can't get it to work. Games still play just fine, but I can only run them from the Steam tray icon, which means I'm limited to the 5 most recent games.

I also tried "small mode" and it didn't produce any better results.

[This is what I get when I run Steam from a terminal](https://gist.github.com/techmouse/6a40151930da47c1db778c33812da099)

To be clear, it's only the main Steam window that's just a black square. Other Steam windows, like chat windows, the settings window, and even the big drop down menus for "Store" and "Library", are visible and working just fine. But it's only by memory and guesswork I can find these things.

#### Steps for reproducing this issue:
1. Install i3 ( `sudo apt-get install i3` ) and log out
2. Select i3 for the session at the login screen, then log in
3. Run Steam

**Comment by kisak-valve** [comment]
Hello @techmouse, this reads like #10543.

Can you test if completely closing steam, then running steam from a terminal with `-system-composer` and/or `-cef-disable-gpu` works around the issue you're seeing?

**Comment by techmouse** [comment]
> Hello @techmouse, this reads like #10543.
> 
> Can you test if completely closing steam, then running steam from a terminal with `-system-composer` and/or `-cef-disable-gpu` works around the issue you're seeing?

Yep, both of those fixed it! Thank you very much! Which one is the preferred option?

**Comment by DrMcCoy** [comment]
Same issue on Englightenment DR16 (e16).

Also, the Steam window now follows me when I switch virtual desktops, which is really annoying.

**Comment by mDuo13** [comment]
I've been seeing the same thing on Arch Linux with Openbox as my window manager. Same as techmouse, using either `-system-composer` or `-cef-disable-gpu` fixes the problem although I'm not sure why exactly or which I should prefer.

**Comment by flan6** [comment]
Disabling gpu acceleration in settings seems to fix it. 

distro: Arch
window manager: i3
gpu: RTX 4070
drivers: nividia 550.78-1

Steam Beta Branch:  Stable Client
Steam Version:  1709846872
Steam Client Build Date:  Wed, Mar 6 17:28 UTC -08:00
Steam Web Build Date:  Thu, Mar 7 18:17 UTC -08:00
Steam API Version:  SteamClient021

**Comment by knightspore** [comment]
I had the same issue and I've also fixed it with either of those flags (would be great to know which is the preferred flag). For some reason, this only started happening to me after a mid-December 2025 update. 

distro: Manjaro
window manager: i3
gpu: RTX 3060TI
drivers: 550.144.03

Steam Beta Branch:  Steam Beta Update
Steam Version:  1739497777
Steam Client Build Date:  Thu, Feb 13 8:29 PM UTC -08:00
Steam Web Build Date:  Fri, Feb 14 1:41 AM UTC -08:00
Steam API Version:  SteamClient021

**Comment by knightspore** [comment]
I just noticed some transparent windows weren't working due to running `picom` (on `i3` launch) with a deprecated flag. I removed the flag which fixed the transparency issue, and I'm now able to run steam as usual.
